shell/pages/home.vue

```diff
@@ -599,6 +611,9 @@ export default defineComponent({
                       :to="createLocation"
                       class="btn btn-sm role-primary"
                       data-testid="cluster-create-button"
+                      role="link"
+                      :aria-label="t('generic.create')"
+                      @keyup.space="$router.push(createLocation)"
```

Comment:
There's quite a few links that still use `space` for activation. I think these can be removed.

Reply:
But this one is a link being used as a button, so in that case I believe it is correct to add `role=button` and to handle the space key.

https://www.w3.org/WAI/ARIA/apg/patterns/button/

Reply:
Right.. I think that it makes more sense to just use a button with a single click event so we don't need to worry about handling edge cases like activating on space. I won't argue the point too much since this is an existing implementation, but I will advocate against promoting future usage of this pattern.